scripts/manual_operations/tests/test_manual_submission.py

```diff
@@ -156,3 +148,130 @@ def test_submit_run(self):
         self.sub_run_args[0].send.assert_called_with('/queue/DataReady',
                                                      message,
                                                      priority=1)
+
+    @patch('icat.Client')
```

Comment:
Shouldn't these be `icat.Client.__init__`? (i.e. mocking the init method rather than class)

Reply:
Doesn't matter in this case as we only care about the raised exception from connect